CHANGELOG.md

```diff
@@ -22,6 +22,9 @@
 * Improve performance of `TrailingNewlineRule`.  
   [Keith Smiley](https://github.com/keith)
 
+* Lint parentheses around switch statements  
```

Comment:
FYI: our format for changelog entries includes a period, followed by two spaces on the last line before the attribution.

So this line should be:

```markdown
* Lint parentheses around switch statements.  
```

Note the trailing two spaces. I know it's strange formatting, but it's what we have... Thanks!